plugins/inputs/statsd/statsd.go

```diff
@@ -831,7 +845,7 @@ func (s *Statsd) handler(conn *net.TCPConn, id string) {
 			b.WriteByte('\n')
 
 			select {
-			case s.in <- b:
+			case s.in <- input{Buffer: b, Time: time.Now(), Addr: conn.RemoteAddr()}:
```

Comment:
On the source tag, it should only have the hostname/ip without the port.  I think this does host:port, so we will need to remove the port.